src/BinaryParsers/PEBinary/ProgramDatabase/CompilerCommandLine.cs

```diff
@@ -128,6 +147,14 @@ public CompilerCommandLine(string commandLine)
                         {
                             this.UsesDebugCRuntime = false;
                         }
+                        else if (argument.EndsWith("GL"))
+                        {
+                            this.WholeProgramOptimization = true;
```

Comment:
>WholeProgramOptimization [](http://example.com/codeflow?start=33&length=24)

Are we missing 'Enabled' on the end of this boolean? As we have with 'EliminateDuplicateStringsEnabled'?

Just looking to be as consistent as possible.

Reply:
That's fair.  Will change.